src/views/Residents.js

```diff
@@ -158,6 +158,7 @@ export default function Residents() {
   const point = need.point(pointCursor);
   const { getResidents, syncResidents } = usePointCache();
 
+  const goBack = useCallback(() => pop(), [pop]);
```

Comment:
Does the indirection here actually change anything? Wild if so.

Reply:
it does, shockingly